.github/workflows/main.yml

```diff
@@ -28,7 +28,15 @@ jobs:
       run: |
         sudo add-apt-repository ppa:ubuntu-toolchain-r/test
         sudo apt update
-        sudo apt-get install -y gcc-9 g++-9 libopenjp2-7-dev libgdal-dev gdal-bin libexpat1-dev libgraphicsmagick++1-dev libnetcdf-dev
+        sudo apt-get install -y gcc-9 g++-9 python3-pip libopenjp2-7-dev libgdal-dev gdal-bin libexpat1-dev libgraphicsmagick++1-dev libnetcdf-dev
```

Comment:
We require python3 now, would it be the same python  version as everywhere else?

Reply:
In CVAT-VSM, Python3 pip3 is only needed for cget, which is needed for installing nlohmann/json.